nixos/lib/test-driver/src/test_driver/machine/ocr.py

```diff
@@ -0,0 +1,110 @@
+import itertools
+import multiprocessing
+import os
+import shutil
+import subprocess
+
+from test_driver.errors import MachineError
+
+
+def perform_ocr_on_screenshot(screenshot_path: str) -> str:
+    return perform_ocr_variants_on_screenshot(screenshot_path, False)[0]
```

Comment:
Do we really need the `perform_ocr_on_screenshot` function, that just passes parameters through in a single line?

Reply:
resolved in offline ocnversation